src/styles/theme/themes/light.ts

```diff
@@ -1,12 +1,15 @@
+import getPlatform from '@libs/getPlatform';
 import colors from '@styles/theme/colors';
 import type {ThemeColors} from '@styles/theme/types';
 import CONST from '@src/CONST';
 import SCREENS from '@src/SCREENS';
 
+const platform = getPlatform();
+
 const lightTheme = {
     // Figma keys
     appBG: colors.productLight100,
-    splashBG: colors.green400,
+    splashBG: platform === CONST.PLATFORM.WEB || platform === CONST.PLATFORM.DESKTOP ? colors.productDark100 : colors.green400,
```

Comment:
I hear ya @ishpaul777 but @parasharrajat is right that in this case we need to use platform files, because that's just the code style we've agreed upon for this repo.